src/main/java/study/step4/model/Movement.java

```diff
@@ -0,0 +1,24 @@
+package study.step4.model;
+
+import java.util.Random;
+
+/**
+ * Movement Class
+ *
+ * 이동을 제한하는 클래스
+ */
+public class Movement {
+    private static final int RANDOM_RANGE = 10;
+    private static final int MOVE_RESTRICT = 4;
+    private static final Random random = new Random();
+
+    public Movement() {
+    }
+
+    public int stopAndMove() {
+        if(random.nextInt(RANDOM_RANGE) >= MOVE_RESTRICT) {
```

Comment:
ealry pattern잘 적용 해주셨네요! :)
추가로 다른 의견도 드려봅니다. 
이번엔 아래의 작성해드린 코드처럼 작성할 수 있다라는 예시만 드리는 겁니다 ! 참고만 부탁드릴게용
아래처럼 분리한 이유는 연산을 하는 부분을 메소드로 분리해 더 명확하게 명시를 하고 읽기 좋은 코드가 될 것 같아 의견 드렸습니다! 
```suggestion
    public int stopAndMove() {
        if (isMove()) {
            return 1;
        }
        return 0;
    }
    
    public boolean isMove() {
        return random.nextInt(RANDOM_RANGE) >= MOVE_RESTRICT;
    }
```

Reply:
넵 감사합니다!